emar.py

```diff
@@ -78,7 +78,7 @@ def run():
           newargs[j] = full_newname
           to_delete.append(full_newname)
           contents.add(newname)
-        except:
+        except IOError:
```

Comment:
I'm not sure exactly why the original author thought it was OK to fail here... so I don't know if IOError is enough to capture their intention.    Shame we dont have a use case or test case or a better comment.

Reply:
Changed to `except Exception:`